### ISSUE TITLE
Synoptic: Multiple beam paths

### ISSUE THREAD

**Issue by KathrynBaker** [ISISComputingGroup/IBEX#1195]
As an IBEX User I would like to be able to split the beam on the synoptic to show multiple paths

**Acceptance Criteria**
1. The beam can be split into n paths

**Notes**
1. This is not a small ticket!
2. This needs discussing to figure out the smaller steps needed, and where such changes would need to occur (e.g. synoptic display, xml schema, synoptic editor)
3. There is almost a work around in that synoptic items can be grouped under each other, but branching the beam would provide a better user experience

**Comment by kjwoodsISIS** [comment]
During Sprint Pre-Planning on 13th June 2016, we noted that this proposal might not be practical.  If the beam gets split into say 3 paths (for the 3 current muon instruments), the resulting synoptic will take up a lot of space.  
Rather than show the muon FE plus the downstream paths, it might be better to create synoptics that show the "upstream" devices on each of the three instruments.

**Comment by KathrynBaker** [comment]
I appreciate, and agree with that thinking, however, championing the scientist who requested this, I think they want to be able to see all the items on one screen, not four, as that requires a lot of recalling of things from one screen to another.

Regards,
Kathryn

**Comment by ChrisM-S** [comment]
Maybe we get them to simply sketch what they would like to see on the screen, perhaps with some awareness of the size of screen they will be using? (keep them in control of the requirements)

I’m sure a synoptic diagram with some active numbers/indicators and clickable items to drill down on is what they want and we have the pieces and technology for this but I’m not sure we should to be too focused on shoe-horning this into a synoptic for a neutron instrument largely based on the requirement of the reflectometers.  A more flexible synoptic is likely to be a gain, particularly with signs of interest we have had from other partners, also instruments like IRIS (in the past) have shown their more complex geometry of the instrument on the control system using a bitmap.

Chris

**Comment by kjwoodsISIS** [comment]
I agree with Chris.  Let's get some sketches of how it _might_ look.

**Comment by ThomasLohnert** [comment]
Muons still want multiple beam paths in their synoptic.

**Comment by DominicOram** [comment]
Requested again by muons

**Comment by KathrynBaker** [comment]
This is still likely to be wanted, whilst we have provided this in OPIs that is not the ideal long term solution (our input is required for OPIs, whilst synoptics can be modified more readily) and so this ticket remains valid.
